src/Toast/sgds-toast.ts

```diff
@@ -0,0 +1,96 @@
+import { customElement, property, state, query } from "lit/decorators.js";
+import { html, literal } from "lit/static-html.js";
+import { ifDefined } from "lit/directives/if-defined.js";
+import { classMap } from "lit/directives/class-map.js";
+import SgdsElement from "../utils/sgds-element";
+import styles from "./sgds-toast.scss";
+
+export type Variant =
+  | "primary"
+  | "secondary"
+  | "success"
+  | "danger"
+  | "warning"
+  | "info"
+  | "dark"
+  | "light";
+
+@customElement("sgds-toast")
+export class SgdsToast extends SgdsElement {
+  static styles = styles;
+
+  @query('[part~="base"]') base: HTMLElement;
+
+  @property({ type: Boolean, reflect: true }) show = true;
+
+  /**
+   * Apply a CSS fade transition to the toast
+   */
+  @property({ type: Boolean, reflect: true }) animation?: boolean;
+
+  @property({ type: Boolean, reflect: true }) autohide? = false;
+
+  @property({ type: Number, reflect: true }) delay = Infinity;
+
+  /** The toast variant. */
+  @property({ type: String, reflect: true }) bg?:
```

Comment:
@haphizi 
theres a missing Variant properties for toast outline for different state. 
You can refer to https://www.designsystem.tech.gov.sg/components/toast